CHANGELOG.md

```diff
@@ -33,6 +33,9 @@
   [426](https://github.com/SwiftGen/SwiftGen/issues/426)
   [468](https://github.com/SwiftGen/SwiftGen/issues/468)
   [573](https://github.com/SwiftGen/SwiftGen/pull/573)
+* Strings: templates to generate Objective-C. Please check [the template's documentation](Documentation/templates/string/objc.md) for more information.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/2c514756ffc950c7f241f261ce41ccadacf7ae87/CHANGELOG.md#L36"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>